src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Packed.cs

```diff
@@ -114,7 +114,7 @@ public static bool Contains(ref short searchSpace, short value, int length)
             {
                 ref short currentSearchSpace = ref searchSpace;
 #pragma warning disable IntrinsicsInSystemPrivateCoreLibAttributeNotSpecificEnough // The else condition for this if statement is identical in semantics to Avx2 specific code
-                if (Avx512BW.IsSupported && Vector512.IsHardwareAccelerated && length > Vector512<short>.Count)
+                if (Avx512F.IsSupported && Avx512BW.IsSupported && Vector512.IsHardwareAccelerated && length > Vector512<short>.Count)
```

Comment:
This should just need `Avx512BW.IsSupported` as it being true implies `Avx512F.IsSupported`. Same with other places below

We might even be able to simplify it down to `Vector512.IsHardwareAccelerated` + `Debug.Assert(Avx512BW.IsSupported)` given that the JIT currently requires `F + BW + CD + DQ + VL` and no other platforms support `V512`

Reply:
> We might even be able to simplify it down to `Vector512.IsHardwareAccelerated` + `Debug.Assert(Avx512BW.IsSupported)` given that the JIT currently requires `F + BW + CD + DQ + VL` and no other platforms support `V512`

I assumed the #pragma warning disable `IntrinsicsInSystemPrivateCoreLibAttributeNotSpecificEnough `and `compExactlyDependsOn` in called method means I need to have the `Avx512F.IsSupported `check.